v22.2/create-external-connection.md

```diff
@@ -0,0 +1,170 @@
+    ~~~
+
+1. To check a list of all external connections, run:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~sql
+    SHOW CREATE ALL EXTERNAL CONNECTIONS;
+    ~~~
+
+    ~~~
+    connection_name |                                                                              create_statement
+    ----------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+    backup_bucket   | CREATE EXTERNAL CONNECTION 'backup_bucket' AS 's3://bucket name?AWS_ACCESS_KEY_ID={access key}&AWS_SECRET_ACCESS_KEY={secret access key}'
+    ~~~
+
+1. Run the backup to your S3 bucket using the external connection's name:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~sql
+    BACKUP DATABASE movr INTO 'external://backup_bucket' AS OF SYSTEM TIME '-10s' WITH revision_history;
+    ~~~
+
+1. Use [`SHOW BACKUP`](show-backup.html) to view your backups in the storage defined by an external connection:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~sql
+    SHOW BACKUPS IN 'external://backup_bucket';
+    ~~~
+    ~~~
+            path
+    ------------------------
+    2022/09/19-134559.68
+    2022/10/10-192044.40
+    (2 rows)
+    ~~~
+
+1. In the event that a restore is necessary, use `RESTORE` with the external connection: 
+
+    {% include_cached copy-clipboard.html %}
+    ~~~sql
+    RESTORE DATABASE movr FROM LATEST IN 'external://backup_bucket';
+    ~~~
+
+1. To delete the external connection:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~sql
+    DROP EXTERNAL CONNECTION backup_bucket;
+    ~~~
+
+## Create an external connection for a changefeed sink
+
+In this example, you create an external connection to a Kafka sink to which a changefeed will emit messages. When you create the external connection, you will include the necessary query parameters for your changefeed. As a result, you will only need to specify the external connection's name when creating a changefeed rather than the Kafka URI and parameters.
+
+1. Define your external connection that references the Kafka sink URI and any [query parameters](changefeed-sinks.html#kafka):
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CREATE EXTERNAL CONNECTION kafka_sink AS 'kafka://broker.address.com:9092?topic_prefix=bar_&tls_enabled=true&ca_cert={certificate}&sasl_enabled=true&sasl_user={sasl user}&sasl_password={url-encoded password}&sasl_mechanism=SASL-SCRAM-SHA-256';
+    ~~~
+
+1. Create your changefeed using the external connection's name:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CREATE CHANGEFEED FOR TABLE movr.users INTO 'external://kafka_sink' WITH resolved;
+    ~~~
+
+## See also
+
+- [`CREATE CHANGEFEED`](create-changefeed.html)
+- [`BACKUP`](backup.html)
+- [`RESTORE`](restore.html)
+- [`EXPORT`](export.html)
+- [`IMPORT INTO`](import-into.html)
```

Comment:
Q: worth adding `DROP EXT` and `SHOW CREATE EXT` here as well? Or already adequately presented earlier ( 👍  )